vfox-restServer/src/main/resources/dbScript/baseV0.1.sql

```diff
@@ -0,0 +1,192 @@
+CREATE TABLE IF NOT EXISTS `vfox`.`UserRole` (
+  `Id` INT NOT NULL AUTO_INCREMENT,
+  `Role` VARCHAR(45) NOT NULL,
+  `RoleState` ENUM('A', 'D', 'I') NOT NULL COMMENT 'A=Active, D= Deleted, I= Inactive',
+  `CreatedBy` INT NULL,
```

Comment:
Let's make CreatedBy, CreatedAt, UpdatedBy, UpdatedAt 'NOT NULL'. I know it's pain while testing but in the long run it's always a good habit.

Reply:
Updated CreatedBy, CreatedAt, UpdatedBy, UpdatedAt  as 'NOT NULL' and committed the same.